apps/general/models.py

```diff
@@ -1,3 +1,230 @@
 from django.db import models
+from django.core.validators import RegexValidator
+from passlib.hash import bcrypt 
+import bcrypt,uuid,base64
 
-# Create your models here.
+def company_logos(instance, filename):
+    company_name = instance.name.replace(' ', '_')
+    unique_id = uuid.uuid4().hex[:6] #Generates a unique identifier
```

Comment:
Where are you using this unique_id and what is the purpose?

Reply:
Here I am using unique_id because it will generate a unique code with 6 letters.
like in media folder it will create another folder with the name of company name + unique code with 6 letters and then inside this folder that file will be stored. So if anyone having duplicate names also we can find it easily with this.